code/game/machinery/computer/arcade.dm

```diff
@@ -1027,3 +1027,285 @@
+		candidate_name = random_name(candidate_gender, initial(wrong_species.name))
+		good_candidate = FALSE
+	else
+		candidate_name = random_name(candidate_gender, initial(cand_species.name))
+
+	if(prob(PROB_CANDIDATE_ERRORS)) // Planet
+		planet_of_origin = pick(incorrect_planets)
+		good_candidate = FALSE
+	else
+		planet_of_origin = pick(planets)
+
+	if(prob(PROB_CANDIDATE_ERRORS)) // Requested Job
+		job_requested = pick(incorrect_jobs)
+		good_candidate = FALSE
+	else
+		job_requested = pick(jobs)
+
+	if(prob(PROB_CANDIDATE_ERRORS)) // Employment records
+		employment_records = pick(incorrect_records)
+		good_candidate = FALSE
+	else
+		employment_records = pick(records)
+
+	if(job_requested == "Clown") // Clowns always get hired no matter what, ZERO requirements
+		good_candidate = TRUE
+
+/obj/machinery/computer/arcade/recruiter/proc/unique_candidate()
+	unique_candidate = pick(UNIQUE_STEVE, UNIQUE_MIME, UNIQUE_CEO_CHILD, UNIQUE_VIGILANTE)
+	switch(unique_candidate)
+		if(UNIQUE_STEVE) // Steve is special
+			candidate_name = "Steve"
+			candidate_gender = MALE
+			age = "30"
+			cand_species = /datum/species/human
+			planet_of_origin = "Unknown"
+			job_requested = "Central Command Intern"
+			employment_records = "Experience in pressing buttons"
+		if(UNIQUE_MIME) // Only hire mimes that don't fill their employment application
+			candidate_name = "..."
+			candidate_gender = "..."
+			age = "..."
+			planet_of_origin = "..."
+			job_requested = "Mime"
+			employment_records = "..."
+		if(UNIQUE_CEO_CHILD) // Hes the son of the CEO, what do you expect?
+			candidate_name = "John Nanotrasen Junior"
+			candidate_gender = MALE
+			age = "12"
+			cand_species = /datum/species/human
+			planet_of_origin = "Unknown"
+			job_requested = "Captain"
+			employment_records = "Whatever"
+		if(UNIQUE_VIGILANTE) // For some reason vigilantes do get inside NT stations, let them slip in
+			candidate_name = "Owlman"
+			candidate_gender = MALE
+			age = "38"
+			cand_species = /datum/species/human
+			planet_of_origin = "Unknown"
+			job_requested = "Assistant"
+			employment_records = "Experience in hunting criminals"
+
+/obj/machinery/computer/arcade/recruiter/proc/win()
+	game_status = RECRUITER_STATUS_START
+	atom_say("Congratulations recruiter, the company is going to have a productive shift thanks to you.")
+	playsound(loc, 'sound/arcade/recruiter_win.ogg', 30)
+	prizevend(50)
+
+/obj/machinery/computer/arcade/recruiter/ui_interact(mob/user, ui_key = "main", datum/tgui/ui = null, force_open = FALSE, datum/tgui/master_ui = null, datum/ui_state/state = GLOB.default_state)
+	ui = SStgui.try_update_ui(user, src, ui_key, ui, force_open)
+	if(!ui)
+		ui = new(user, src, ui_key, "NTRecruiter", name, 400, 480)
+		ui.open()
+
+/obj/machinery/computer/arcade/recruiter/ui_data(mob/user)
+	var/list/data = list(
+	"gamestatus" = game_status,
```

Comment:
Can you indent the contents of this list by one level?

Reply:
Done